src/NTDll/NTDll+PROCESS_BASIC_INFORMATION.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="PROCESS_BASIC_INFORMATION"/> nested type.
+    /// </content>
+    public static partial class NTDll
+    {
+        /// <summary>
+        /// Contains basic information about a process.
+        /// </summary>
+        /// <seealso href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms684280(v=vs.85).aspx"/>
+        [StructLayout(LayoutKind.Sequential)]
+        public struct PROCESS_BASIC_INFORMATION
+        {
+            /// <summary>
+            /// This value is reserved.
+            /// </summary>
+            public IntPtr Reserved1;
+
+            /// <summary>
+            /// The base address of the PEB structure in the process memory.
+            /// </summary>
+            public IntPtr PebBaseAddress;
```

Comment:
Since changing this from `IntPtr` to `PEB*` later will be a binary breaking change, can we do so now? This will involve defining the `PEB` structure. Let me know if you have any trouble doing that.

Reply:
[The format of the `PEB` structure is different for 32-bit windows and 64-bit Windows](https://docs.microsoft.com/en-us/windows/win32/api/winternl/ns-winternl-peb#remarks). How do you usually handle such structs?

Reply:
Eww.. We know how to handle pointer size differences but that looks like a totally different layout. Well, we could define both and leave folks to cast the pointer to the right one, or we could define a ref assembly that only has the common fields and two per-architecture runtime assemblies to implement the right one for each. Both are a bit icky. The first is easier while the second one is probably the best world.
I don't have time for the latter option just now, so if you can figure it out, great. Otherwise the first option may be what we must go with.
I guess since the start out similarly, another option is to only define the first part as far as the two are common.

Reply:
I left it as void* at the moment; hope that works for you.